docs/ref/models/fields.txt

```diff
@@ -1150,6 +1150,44 @@ when :attr:`~django.forms.Field.localize` is ``False`` or
     information on the difference between the two, see Python's documentation
     for the :mod:`decimal` module.
 
+``GeneratedField``
+------------------
+
+.. class:: GeneratedField(expression, output_field=None, db_persist=True, **kwargs)
+
+A field which is computed based on other fields in the model. Uses the database's
+``GENERATED ALWAYS AS`` syntax. Similar to a database view, this field is managed
+and updated by the database itself.
+
+.. attribute:: GeneratedField.expression
+
+    An :class:`Expression` used by the database to automatically set the field
+    value whenever the model is changed.
+
+    Several restrictions apply to generated columns, depending on the database.
+    A :class:`GeneratedField` should be immutable and only reference
+    concrete columns inside the model itself. References to other models' fields
+    is not supported. See your database's documentation for more information.
+
+.. attribute:: GeneratedField.output_field
+
+    An optional :class:`Field` used to explicitly define the model's underlying
+    column type. This can be used to customize the field's DB collation or other
+    field-specific properties.
+
+    By default, this field is derived from the given ``expression``.
+
+.. attribute:: GeneratedField.db_persist
+
+    Determines if the database column should occupy storage as if it were a real
+    column. If ``False``, the column acts as a virtual column and does
+    not occupy database storage space.
+
+    Default is ``True``. PostgreSQL only supports persisted columns.
+
+``GeneratedField`` is supported on MariaDB, MySQL, PostgreSQL, and SQLite. SQLite
+does not support parameters in generated column definitions.
```

Comment:
I think this should go before the attribute description?

Reply:
I agree